io/eolearn/tests/test_raster_io.py

```diff
@@ -365,8 +365,7 @@ def test_time_dependent_feature_with_timestamps(test_eopatch):
 @pytest.mark.parametrize("no_data_value, data_type", [(np.nan, float), (0, int), (None, float), (1, np.byte)])
 def test_export_import_sequence(no_data_value, data_type):
     """Tests import and export tiff tasks on generated array with different values of no_data_value."""
-    eopatch = EOPatch()
-    eopatch.bbox = BBox((0, 0, 1, 1), crs=CRS.WGS84)
+    eopatch = EOPatch(bbox=BBox((0, 0, 1, 1), crs=CRS.WGS84))
```

Comment:
this repeats quite a lot. would it make sense to have a fixture for some empty eopatch instead?

Reply:
ok, let's leave it like it is for now then